### PR TITLE
[devtools-translate] Russian - What's New in DevTools (Chrome 99)

### LINKED ISSUE #2152: [devtools-translate] Russian - What's New in DevTools (Chrome 99)
Post: https://developer.chrome.com/blog/new-in-devtools-99/

Translator: @solarrust
Reviewers: @webmaxru, @kateryna-prokopenko

Translated by: Feb 25

### PR DESCRIPTION
**Pull request by solarrust**
Fixes #2152 

It was hard this time 🥲
Not sure that all my translations are right and elegant.
My biggest challenge — «Touch-friendly color-picker and split pane» header. 
@kateryna-prokopenko @webmaxru will be glad to hear your opinions 🙏

Peace! 🕊

### CODE REVIEW

File: site/ru/blog/new-in-devtools-99/index.md [hunk 1]
```diff
@@ -6,190 +6,238 @@ authors:
...
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/Wl1pPVfQ51NaCtpp3KuY.png", alt="Улучшенное дерево папок источников во вкладке Источники", width="800", height="444" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/68613ab02f6d556a2c5ac68ea08f466a534c6bd9 #}
 
-Chromium issue: [1284737](https://crbug.com/1284737)
+Задача в трекере Chromium: [1284737](https://crbug.com/1284737)
 
 
 <!-- ### Display worker source files in the Sources panel {: #worker-sourcemap } -->
+### Отображение исходных файлов сервис-воркеров во вкладке Источники (Sources) {: #worker-sourcemap }
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-99/index.md:171]:
```suggestion
### Отображение исходных файлов воркеров во вкладке Источники (Sources) {: #worker-sourcemap }
```

File: site/ru/blog/new-in-devtools-99/index.md [hunk 2]
```diff
@@ -6,190 +6,238 @@ authors:
...
 
 <!-- [Worker](https://web.dev/workers-overview/) (e.g. web worker, service worker) source files with relative SourceURL are now displayed in the **Source** panel. Previously, worker source files were not handled correctly. -->
+Исходные файлы [воркера](https://web.dev/workers-overview/) (веб-воркера, сервис-воркера) со связанными SourceURL теперь отображаются во вкладке **Источники** (Sources). Ранее исходные файлы воркеров обрабатывались некорректно.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-99/index.md:174]:
```suggestion
Исходные файлы [воркера](https://web.dev/workers-overview/) (веб-воркера, сервис-воркера) с относительным SourceURL теперь отображаются во вкладке **Источники** (Sources). Ранее исходные файлы воркеров обрабатывались неправильно.
```

File: site/ru/blog/new-in-devtools-99/index.md [hunk 3]
```diff
@@ -6,190 +6,238 @@ authors:
...
 
-Chromium issue: [1277002](https://crbug.com/1277002)
+Задача в трекере Chromium: [1277002](https://crbug.com/1277002)
 
 
 <!-- ## Chrome’s Auto Dark Theme updates {: #auto-dark-mode } -->
+## Обновление автоматической тёмной темы Chrome {: #auto-dark-mode }
 
 <!-- The [Auto Dark Theme emulation](/blog/new-in-devtools-96/#auto-dark-mode) UI is now simplified. It is a checkbox now, it was a dropdown previously. -->
+UI [эмуляции автоматической тёмной темы](/blog/new-in-devtools-96/#auto-dark-mode) теперь упрощён. Раньше это был выпадающий список, теперь это чекбокс.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-99/index.md:188]:
```suggestion
UI [эмуляции автоматической тёмной темы](/blog/new-in-devtools-96/#auto-dark-mode) теперь упрощён. Сейчас это чекбокс, раньше это был выпадающий список.
```

File: site/ru/blog/new-in-devtools-99/index.md [hunk 4]
```diff
@@ -6,190 +6,238 @@ authors:
...
 
 <!-- Apart from that, when the [Auto Dark Theme](/blog/auto-dark-theme/) is enabled, the **Emulate prefers-color-scheme** dropdown will be disabled and set to **prefers-color-scheme: dark** automatically. -->
+Кроме того, если активируется [автоматическая тёмная тема](/blog/auto-dark-theme/), то выпадающий список **Эмулировать едиафункцию prefers-color-scheme** становится неактивным и устанавливает значение **prefers-color-scheme: dark** автоматически.
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-99/index.md:191]:
```suggestion
Кроме того, если [автоматическая тёмная тема](/blog/auto-dark-theme/) активирована, выпадающий список **Эмулировать медиафункцию prefers-color-scheme** становится неактивным и устанавливает значение **prefers-color-scheme: dark** автоматически.
```

File: site/ru/blog/new-in-devtools-99/index.md [hunk 5]
```diff
@@ -6,190 +6,238 @@ authors:
...
+{% Img src="image/dPDCek3EhZgLQPGtEG3y0fTn4v82/eqfY1jZI8kY7BknnuAom.png", alt="Эмуляция автоматической тёмной темы", width="800", height="476" %}
 
 {# https://chromium.googlesource.com/devtools/devtools-frontend/+/8443d2894b6401695ce94657e6afd5ad399eef28 #}
 
-Chromium issue: [1243309](https://crbug.com/1243309)
+Задача в трекере Chromium: [1243309](https://crbug.com/1243309)
 
 
 <!-- ## Touch-friendly color-picker and split pane {: #touch-friendly } -->
+## Выбор цвета и разделитель панелей, отзывчивые на устройствах с сенсорным экраном {: #touch-friendly }
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-99/index.md:205]:
```suggestion
## Удобные сенсорные палитра цветов и разделитель панелей {: #touch-friendly }
```

File: site/ru/blog/new-in-devtools-99/index.md [hunk 6]
```diff
@@ -6,190 +6,238 @@ authors:
...
 <!-- ## Miscellaneous highlights {: #misc } -->
+## Другие важные моменты {: #misc }
 
 <!-- These are some noteworthy fixes in this release: -->
+Вот несколько заслуживающих внимания исправлений в этом выпуске:
 
 <!-- - Fixed the [edit cookies](/docs/devtools/storage/cookies/#edit) issue in the **Cookies** pane. ([1290196](https://crbug.com/1290196)) -->
+- Исправлены неполадки с [редактированием cookies](/docs/devtools/storage/cookies/#edit) в панели **Файлы cookie** (Cookies). ([1290196](https://crbug.com/1290196))
 <!-- - Use `Shift` + `Tab` to select the previous command in the [Command menu](/docs/devtools/command-menu/). ([1278743](https://crbug.com/1278743)) -->
+- Использование `Shift` + `Tab для выбора предыдущей команды в [Командном меню (Command menu)](/docs/devtools/command-menu/). ([1278743](https://crbug.com/1278743))
```
Comment (kateryna-prokopenko) [inline site/ru/blog/new-in-devtools-99/index.md:230]:
```suggestion
- Использование `Shift` + `Tab` для выбора предыдущей команды в [Командном меню (Command menu)](/docs/devtools/command-menu/). ([1278743](https://crbug.com/1278743))
```

**Review by kateryna-prokopenko — COMMENTED** [review]
@solarrust thanks for the translation! Added some comments

File: site/ru/blog/new-in-devtools-99/index.md [hunk 7]
```diff
@@ -6,190 +6,238 @@ authors:
...
 
-Chromium issue: [1068788](https://crbug.com/1068788)
+Задача в трекере Chromium: [1068788](https://crbug.com/1068788)
 
 
 <!-- ## Sourcemaps improvements {: #sourcemap } -->
+## Улучшение карты источников {: #sourcemap }
 
 <!-- ### Debug Chrome extension with sourcemap files {: #extension } -->
+### Отладка расширений Chrome с помощью файлов исходных карт {: #extension }
```
[the remark this answers is not in the document]
Comment (solarrust) [reply site/ru/blog/new-in-devtools-99/index.md:145]: We already used translation «карта источников» [there](https://developer.chrome.com/ru/blog/new-in-devtools-100/#misc). Seems like better variant. But I'll be glad to see alternatives is they are 🙏

File: site/ru/blog/new-in-devtools-99/index.md [hunk 8]
```diff
@@ -6,190 +6,238 @@ authors:
...
 
 <!-- You can now [debug Chrome extension](/docs/extensions/mv3/getstarted/#unpacked) with sourcemap files. Previously, DevTools only supported inline sourcemap for Chrome extension debugging. -->
+Теперь вы можете [отлаживать расширения для Chrome](/docs/extensions/mv3/getstarted/#unpacked) с использованием файлов исходных карт. Ранее DevTools поддерживал только встроенную карту источников для отладки расширений Chrome.
```
[the remark this answers is not in the document]
Comment (solarrust) [reply site/ru/blog/new-in-devtools-99/index.md:148]: Answered this question in prev comment 🙌

**Review by solarrust — COMMENTED** [review]
Thank you a lot for review @kateryna-prokopenko 👍

**Comment by solarrust** [comment]
@webmaxru the ball's in your court 🙌

**Comment by solarrust** [comment]
@jecfish can you help me please? Something went wrong with this PR.

**Comment by solarrust** [comment]
@jecfish friendly reminder for help! 🙏

**Comment by jecfish** [comment]
Reopen the request! I thought it is merged. Sorry for late reply.

**Comment by jecfish** [comment]
Ping @webmaxru for review

**Comment by jecfish** [comment]
Due to the conflicts and stale branch, created a new PR instead #2759

**Pull request closed without merging**